app.rb

```diff
@@ -1,20 +1,174 @@
 require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
+
 apartments = data[:apartments]
 tenants = data[:tenants]
+ruby_apartments = []
+ruby_tenants = []
+
+apartments.each do |apartment|
```

Comment:
Could do `ruby_apartments = apartments.map{ ... }` so you don't need to worry about pushing inside of `.each`

Same deal with `tenants.map{ ... }`